server/repositories/accounts.py

```diff
@@ -28,7 +28,6 @@ class Account(TypedDict):
 
 
 async def create(
-    account_id: int,
```

Comment:
account_id is an auto incrementing int; no need to create in application code

Reply:
(it actually makes more sense from a scale perspective to allow the database to ensure these are consistent across all replicas)

Reply:
matters less with a UUID since it's very much random & non-sequential